lib/rides/pages/search_ride_page.dart

```diff
@@ -196,12 +204,16 @@ class _SearchRideFormState extends State<SearchRideForm> {
           children: [
             AddressSearchField.start(
               controller: _startController,
-              onSelected: (suggestion) {},
+              onSelected: (suggestion) {
+                _startSuggestion = suggestion;
```

Comment:
Can you add read only? So we can fix the bug with being able to write without the Addressuggestion

Reply:
nevermind apparently readOnly does not solve our issue. Since we tried that alread in CreateDrive